retrieval/series_cache.go

```diff
@@ -100,6 +100,10 @@ type seriesCacheEntry struct {
 	suffix   string
 	hash     uint64
 
+	// Value of the most recent point seen for the time series. If a new value is
+	// less than the previous, then the series has reset.
+	previousValue float64
+
 	hasReset       bool
 	resetValue     float64
 	resetTimestamp int64
```

Comment:
Want to take the opportunity to add a blank line after this one? Or is this how the auto-formatter chose to format it?

Reply:
good idea -- added some additional comments too